stable/redis-ha/templates/redis-sentinel-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: {{ template "fullname" . }}-sentinel-svc
+    role: service
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
```

Comment:
There are a standard set of labels to use by convention and a helper function for use for this boilerplate. 

1. Please use https://github.com/kubernetes/charts/blob/master/incubator/istio/templates/_helpers.tpl as your helpers file contents
2. Update manifests to use the function. Example here: https://github.com/kubernetes/charts/blob/master/incubator/istio/templates/addons-grafana-deployment.yaml#L8